src/parenthesis.ts

```diff
@@ -67,19 +63,22 @@ export class Parenthesis extends Modifier {
     super();
 
     this.position = position ?? Modifier.Position.LEFT;
+    if (this.position == Modifier.Position.RIGHT) {
+      this.text = '\uE0F6'; /*noteheadParenthesisRight*/
+    } else if (this.position == Modifier.Position.LEFT) {
+      this.text = '\uE0F5'; /*noteheadParenthesisLeft*/
+    }
 
-    this.point = Tables.currentMusicFont().lookupMetric('parenthesis.default.point') ?? Note.getPoint('default');
-    this.setWidth(Tables.currentMusicFont().lookupMetric('parenthesis.default.width'));
+    this.setFontSize(Note.getPoint('default'));
   }
 
   /** Set the associated note. */
   setNote(note: Note): this {
     this.note = note;
-    this.point = Tables.currentMusicFont().lookupMetric('parenthesis.default.point') ?? Note.getPoint('default');
-    this.setWidth(Tables.currentMusicFont().lookupMetric('parenthesis.default.width'));
     if (isGraceNote(note)) {
-      this.point = Tables.currentMusicFont().lookupMetric('parenthesis.gracenote.point') ?? Note.getPoint('gracenote');
-      this.setWidth(Tables.currentMusicFont().lookupMetric('parenthesis.gracenote.width'));
+      this.setFontSize(Note.getPoint('gracenote'));
+    } else {
+      this.setFontSize(Note.getPoint('default'));
```

Comment:
Why isn't it named Note.getFontSize()?

Reply:
Very good idea! rather get the font from the note: note.getFont()

Reply:
And remove note.getPoint